app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,160 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch all forms under the admin access.
```

Comment:
It looks like if there's no "title" defined, then the description is truncated to 50 characters and used as the title. Since this doesn't really offer any new information, would it be better to set the "title" for each endpoint to what it was in the old docs?

**Old:**
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/d8236d55-02f5-4f17-9cc6-c449e0159b17)

**New:**
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/d91ea732-eb93-4158-b757-a4246865f9de)

Reply:
(In other words, would it be better to do:
```
*   title: List all forms in the system
*   description: This endpoint will fetch all forms under the admin access.
```

Reply:
(what's the difference between "title" and "summary" in the OpenAPI spec? Maybe it should be summary instead of title?)

Reply:
Fix